staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -69,6 +73,7 @@ func main() {
 			if err != nil {
 				panic(err)
 			}
+			kind_created_here = true
```

Comment:
In my PR I have added a sleep instead of passing this flag: https://github.com/kubernetes/kubernetes/pull/45463/files#diff-eb875333b11b3ab1e7e6fdb35d1bc09dR75

While sleeping could look hacky, I think it's okay for the example (with the appropriate comment, of course), and it doesn't require to run the example twice.
In your case the first run of example will always fail.